jmx2ini/jmx2ini.go

```diff
@@ -0,0 +1,306 @@
+package main
+
+import (
+	"fmt"
+	"launchpad.net/xmlpath"
```

Comment:
I noticed I can 'go get' the more recent version of this package without any Bazaar dependancy.
go get gopkg.in/xmlpath.v2

Maybe it is a drop-in replacement for  "launchpad.net/xmlpath" ?